src/components/JoinButton.jsx

```diff
@@ -0,0 +1,25 @@
+const JoinButton = () => {
+  // Inline style for the button
+  const buttonStyle = {
+    backgroundColor: "#f0c14b",
+    color: "black",
+    padding: "10px 20px",
+    border: "none",
+    borderRadius: "5px",
+    cursor: "pointer",
+    textDecoration: "none",
+  };
+
+  return (
+    <a
+      href="/join" // Link to the join page
+      style={buttonStyle}
+      onMouseOver={(e) => (e.currentTarget.style.opacity = 0.7)} // Opacity reduced when hovering over
```

Comment:
you can use the hover property for tailwind 

https://tailwindcss.com/docs/hover-focus-and-other-states#hover-focus-and-active

Reply:
Please make sure to also assign the issue to yourself, and read carefully the example I provide in your issue.